wordstats/cognate_info.py

```diff
@@ -1,29 +1,21 @@
-import codecs
-import os
 from datetime import datetime
 
 from python_translators.translators.glosbe_translator import Translator
-from python_translators.translators.glosbe_over_tor_translator import GlosbeOverTorTranslator
```

Comment:
@joelgrondman  - note that many of these removed imports are done automatically by Python when refactoring code since they are not used in the corresponding file. 

For SQLAlchemy I have had in the past code which requires a class to be loaded, even if it's not used straight away in that file. I hope this is not the case anywhere here!